media/css/products/vpn/download.scss

```diff
@@ -301,6 +302,22 @@ $vpn-download-mq-2xl: '(min-width: 1450px)';
     }
 }
 
+.vpn-download-previous-versions {
+    display: flex;
+    justify-content: center;
+    padding-bottom: $layout-lg;
+    padding-top: 0;
+
+    a {
+        color: $color-black;
+        font-weight: 700;
+    }
+
+    @media #{$mq-md} {
```

Comment:
Nit: worth changing the break point slightly for this?

```suggestion
    @media #{$vpn-download-mq-2xl} {
```

It might look nicer centered a bit earlier?

![image](https://user-images.githubusercontent.com/400117/233318285-33f7d0bd-0ebd-44bb-8218-c1f3113ddd05.png)